### ISSUE TITLE
Create sets for laravel upgrade rules

### ISSUE THREAD

**Issue by hirenkeraliya** [rectorphp/rector-laravel#38]
I check all the rules and upgrade for the laravel 7 to 8 and 8 to 9.
I got the rules can we make a set for this laravel upgrade?
if you agree then I will do that.

**Comment by samsonasik** [comment]
The set seems exists until laravel 8

https://github.com/rectorphp/rector-laravel/blob/6bee4284d50e84eb5d027ff266801b2a1c903a33/src/Set/LaravelLevelSetList.php#L64

feel free to create new set list PR until laravel 9

**Comment by TomasVotruba** [comment]
Hi, that would be very useful!

Please go for it :+1:

**Comment by TomasVotruba** [comment]
Hi, that would be very useful!

Please go for it :+1:

**Comment by hirenkeraliya** [comment]
@samsonasik oh it's great. do we have documentation for it anywhere? if no then we also need to do this first.

**Comment by samsonasik** [comment]
@hirenkeraliya there is documentation for use set https://github.com/rectorphp/rector-laravel#use-sets , feel free to create documentation improvement PR.

**Comment by hirenkeraliya** [comment]
@samsonasik 

Initially, I added a [new rector class](https://github.com/rectorphp/rector-laravel/pull/37) at that time I did it using the [package development](https://radicalloop.medium.com/test-your-laravel-package-locally-in-your-laravel-project-dec120686695) blog post. 

But, I tried it today it's not working. And, also the rector configuration is changed. So, how can we debug when we want to print the object like `$methodCall`?
